backend/document/domain/assembly_strategies/assembly_strategy_utils.py

```diff
@@ -12,6 +12,8 @@
     TNBook,
     TQBook,
     USFMBook,
+    RGBook,
+    render_chapter,
```

Comment:
I recommend giving `render_chapter` a more specific name, such as "render_rgbook_chapter`, or else refactoring it into its own module.